python_orocos_kdl/CMakeLists.txt

```diff
@@ -14,20 +14,37 @@ if(DEFINED ENV{ROS_PYTHON_VERSION})
 else()
   SET(PYTHON_VERSION 2 CACHE STRING "Python Version")
 endif()
+
 find_package(PythonInterp ${PYTHON_VERSION} REQUIRED)
 find_package(PythonLibs ${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR} REQUIRED)
 execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib(plat_specific=True, prefix=''))" OUTPUT_VARIABLE PYTHON_SITE_PACKAGES OUTPUT_STRIP_TRAILING_WHITESPACE)
-list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
-find_package(SIP REQUIRED)
-include(SIPMacros)
+set(PYTHON_SITE_PACKAGES_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/${PYTHON_SITE_PACKAGES})
+set(LIBRARY_NAME "PyKDL")
 
-include_directories(${SIP_INCLUDE_DIR} ${PYTHON_INCLUDE_DIRS})
+option(BUILD_PYKDL_PYBIND11 "Use PyBind11 instead of SIP" ON)
```

Comment:
Does this make pybind11 the default wrapper?

Reply:
Yes, PyBind11 is niet the default

Reply:
Feels like a `deprecation` warning is in place for this release?
I think its important to be clear about the path fwd; perhaps it's an OK idea to have an additional in transition release, but if you can afford a hard break, its probably best to take advantage in doing so. Take a clear choice and stand behind it ;)